packages/now-next/test/fixtures/23-custom-routes-verbose/next.config.js

```diff
@@ -37,6 +37,10 @@ module.exports = {
           source: '/to-hello',
           destination: '/hello',
         },
+        {
+          source: '/(.*)-:id(\\d+).html',
+          destination: '/blog/:id',
```

Comment:
I didn't think we supported named groups mixed with unnamed groups?

Reply:
We should probably support it since it does appear to be a valid source and we handle this in Next.js. It looks like the error was caused from not indexing the un-named segments since we were just filtering them which I updated to resolve the added test failure. 

If we don't want to support this we should probably add an error when we detect them being mixed and make sure we match this in Next.js also